### ISSUE TITLE
Teleporting to dungeon/entering BG while in combat exploit

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5472]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
If you are in combat and your dungeon finder pops or you try to teleport inside through the green eye from minimap you shouldn't be allowed to enter untill you leave combat, same should happen for when battleground pop and you try to accept, it should only let you enter when you are out of combat. This can be exploited by players so they enter dungeon/bg when they are about to get killed.
**Expected behaviour**: Tell us what should happen instead
You should only be allowed to teleport inside dungeon/battleground when out of combat.
**Steps to reproduce the problem**:
Get your dungeon/BG pop and try to accept invitation or teleporting inside through the minimap button and you will see how you can enter while you are in combat.
1. 
2. 
3. 

**Include proofs for this behaviour**
I can't find you any proof on the internet about this unless I dig deep into private server forums to find threads about this issue since this wasn't an issue on retail. But anyone with an account on retail could test it there and see that you can't do it.
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/5470
